transport/UdpEndpointImpl.cpp

```diff
@@ -75,7 +39,7 @@ void UdpEndpointImpl::sendStunTo(const transport::SocketAddress& target,
     uint64_t timestamp)
 {
     auto* msg = ice::StunMessage::fromPtr(data);
-    if (msg->header.isRequest() && !_dtlsListeners.contains(target) && !_iceResponseListeners.contains(transactionId))
+    if (msg->header.isRequest() && !_iceResponseListeners.contains(transactionId))
```

Comment:
I turned to relying only on transaction ids for response routing. It is the right way.